ui/src/app/pages/appAnalysis/app-selector.tsx

```diff
@@ -34,11 +36,11 @@ const enum VisibleModal {
   Jupyter = 'Jupyter',
 }
 
-interface AppSelectorProps {
+interface AppSelectorProps extends NavigationProps {
   workspace: WorkspaceData;
 }
 
-export const AppSelector = (props: AppSelectorProps) => {
+export const AppSelector = withNavigation((props: AppSelectorProps) => {
```

Comment:
can use this instead of the HOC:
`const [navigate] = useNavigation();`

Reply:
updated, i was tempted to use currentWorkspaceStore as well rather than passing workspace info as props. But that could be a follow up clean up PR